config/vagrant/Vagrantfile

```diff
@@ -6,7 +6,7 @@ Vagrant.configure('2') do |config|
   config.vm.network 'forwarded_port', guest: 6443, host: 6443
 
   config.vm.define 'k8shost' do |k8shost|
-    k8shost.vm.box = 'generic/fedora36'
+    k8shost.vm.box = 'generic/fedora37'
```

Comment:
Why not fedora39?

Reply:
It doesn't exist.

https://app.vagrantup.com/generic/boxes/fedora39